06.wc/wc.rb

```diff
@@ -3,22 +3,21 @@
 require 'optparse'
 
 def main
-  options = check_options_exist
-  files = make_source_files
-  outputs = count_outputs(files)
+  files_with_extention = select_files_with_extension
```

Comment:
extension（拡張子）の有無がこのプログラムの動作に何か影響を与えるかというと、特にそういう要件はない気がします。
なので、extensionという単語は使わず、

```suggestion
  files = select_files
```

ぐらいでいいかもしれません